language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/annotation_access_ctx_config8.json

```diff
@@ -5,14 +5,6 @@
   },
   "source": "expression_context/source/annotation_access_ctx_source8.bal",
   "items": [
-    {
```

Comment:
Any reason to remove the `OBJECT_FIELD` attach point from the deprecated annotation?

![capture1](https://user-images.githubusercontent.com/35211477/191171493-f096e5f0-9245-4f30-8cc4-5036798d1052.jpeg)

previously we had `OBJECT_FIELD`

![capture 2](https://user-images.githubusercontent.com/35211477/191171566-28f7078a-900b-4e1c-8a02-f41056db7bde.jpeg)

Reply:
Please refer https://github.com/ballerina-platform/ballerina-lang/pull/37178#discussion_r974894013